scaleway/resource_compute_instance_server.go

```diff
@@ -261,7 +263,7 @@ func resourceScalewayComputeInstanceServerUpdate(d *schema.ResourceData, m inter
 }
 
 func resourceScalewayComputeInstanceServerDelete(d *schema.ResourceData, m interface{}) error {
```

Comment:
So we agree that `ServerActionTerminate ` don't remove any volumes?

Reply:
It does remove the root volume!